assets/js/components/tree.js

```diff
@@ -63,6 +63,7 @@ class TreeCollection {
         this._tree         = tree;
         this._el           = $(el);
         this._toggle       = this._el.find('> [data-role="toggle"]');
+        this._toggleBtn    = this._toggle.find('button');
```

Comment:
I think this event does not fire with keyboard operation.
https://github.com/frctl/mandelbrot/pull/106/files#diff-2b41cea9b977c28daf848e97ff2a213fL68

so I suggestoin to change like this and adjust following change.
```suggestion
        this._toggle       = this._el.find('button');
```

For my project, I simply changed the heading tag to a button tag.
![image](https://user-images.githubusercontent.com/13148112/80301361-accfa400-87de-11ea-882b-7e892e77a273.png)